bitcoin-explained/breaking-down-taproot-activation-options.md

```diff
@@ -0,0 +1,1674 @@
+---
+title: "Breaking Down Taproot Activation Options"
+transcript_by: Sjors via rev.com
+revision_by: edilmedeiros
+media: https://youtu.be/mT0t8Jm0m5E?si=ITfvLBQkhA6KA7DG
+tags: ["bitcoin-core", "taproot", "soft-fork"]
+speakers: ["Sjors Provoost","Aaron van Wirdum"]
+categories: ["podcast"]
+date: 2020-07-25

```

Comment:
since today we also support episode numbers, so you can also include the episode number here
```suggestion
date: 2020-07-25
episode: 3
```